source/Jobbr.Server.UnitTests/Jobbr.Server.UnitTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
I'd drop these two options for now and maybe make a decisions on the language level and language features overall at a later point